service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnosticreport/ObservationMapper.java

```diff
@@ -269,6 +269,13 @@ private String mapObservationToObservationStatement(MultiStatementObservationHol
         return Optional.empty();
     }
 
+    private CommentType prepareCommentType(Observation observation) {
+        return observation.getCode().getCoding()
```

Comment:
There is already a method `CodeableConceptMappingUtils.hasCode()`. Maybe you could use this one instead?

Reply:
Good point, done, thanks